homework/index.js

```diff
@@ -30,7 +30,47 @@
   }
 
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const li = createAndAppend('li', ul, {
+      class: 'container',
+    });
+
+    if (repo.name != null) {
+      createAndAppend('li', li, {
```

Comment:
An `li` tag can only have a `ul`, `ol` or `menu` tag as its parent. You can not define as a child of another `li`. This will result in invalid HTML. The browser will try and make the best of it anyway, but it is still an error. This is the HTML you generate:

```html
<div id="root">
  <h2>HYF Repositories</h2>
  <ul>
    <li class="container">
      <li class="leftHandSide">Repository :</li>
      <li class="rightHandSide"><a href="https://github.com/HackYourFuture/alumni" target="_blank">alumni</a></li>
      <li class="leftHandSide">Description :</li>
      <li class="rightHandSide">Alumni Guides, everything you need to land a job</li>
      <li class="leftHandSide">Forks :</li>
      <li class="rightHandSide">10</li>
      <li class="leftHandSide">Updated at :</li>
      <li class="rightHandSide">2019-06-21T11:38:21Z</li>
    </li>
    <li class="container">
      <li class="leftHandSide">Repository :</li>
      <li class="rightHandSide"><a href="https://github.com/HackYourFuture/angular" target="_blank">angular</a></li>
      <li class="leftHandSide">Description :</li>
      <li class="rightHandSide">Angular</li>
      <li class="leftHandSide">Forks :</li>
      <li class="rightHandSide">3</li>
      <li class="leftHandSide">Updated at :</li>
      <li class="rightHandSide">2018-04-06T17:17:51Z</li>
    </li>
    ...
  </ul>
</div>
```

Perhaps I can suggest an HTML `table` instead?